lametro/templates/lametro/event.html

```diff
@@ -5,6 +5,17 @@
 {% load staticfiles %}
 {% block title %}{{event.name}}{% endblock %}
 {% block content %}
+    <div class="container-fluid">
+        <br />
+        <div class="alert alert-danger" role="alert">
+            <i class="fa fa-fw fa-exclamation-circle" aria-hidden="true"></i>
+            * Consistent with AB361, Board and Committee Meetings will continue to meet virtually so long as there’s a declared state of emergency. <a class="alert-link" href="https://leginfo.legislature.ca.gov/faces/billTextClient.xhtml?bill_id=202120220AB361" target="_blank">Click here</a>.
+        </div>
+        <div class="alert alert-danger" role="alert">
+            <i class="fa fa-fw fa-exclamation-circle" aria-hidden="true"></i>
+            De acuerdo con AB361, las reuniones de la Junta Directiva y el Comité continuarán llevándose a cabo virtualmente mientras exista una declaración del estado de emergencia. <a class="alert-link" href="https://leginfo.legislature.ca.gov/faces/billTextClient.xhtml?bill_id=202120220AB361" target="_blank">Haga clic aquí</a>.
```

Comment:
I usually like to include a little arrow with links like this.

```suggestion
            De acuerdo con AB361, las reuniones de la Junta Directiva y el Comité continuarán llevándose a cabo virtualmente mientras exista una declaración del estado de emergencia. <a class="alert-link" href="https://leginfo.legislature.ca.gov/faces/billTextClient.xhtml?bill_id=202120220AB361" target="_blank">Haga clic aquí &raquo;</a>.
```

Reply:
ooh I always love learning these html character codes. consider it done!